gslib/commands/mb.py

```diff
@@ -125,7 +125,7 @@
   retention policy.
 
 <B>UNIFORM BUCKET-LEVEL ACCESS</B>
-  You can specify one of the available settings for a bucket
+  You can specify turn on uniform bucket-level access for a bucket
```

Comment:
Suggestion: "You can enable or disable uniform bucket-level access for a bucket with the -b option"

Reply:
Done